docs/guides/breakings/v3.18.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: Guides.Breakings.V3_18
+title: V3.18
+---
+
+# V3.18
+
+Among the changes in V3.18 was components V2, while splendid, it did bring with it a few changes you should be aware of.
+
+- IMessage
+
+Messages components property has changed. it is now a MessageComponent type. Effectively what that means for Components V2 is that you can simply call Build() on an instance of ComponentBuilderV2 like so:
+```cs
+await RespondAsync(components: cbv2.Build()); // cbv2 is a ComponentBuilderV2 object
+```
+
+- Nested components
+
+The usage of components with `ActionRow` and `ComponentBuilder` has a type change as well. You can find examples in [Components_V2_Advanced].
```

Comment:
```suggestion
The usage of components with `ActionRowBuilder` and `ComponentBuilder` has a type change as well. You can find examples in [Components_V2_Advanced].
```

Reply:
Similart to above, 
`List<IMessageComponent> Components` is now `List<IMessageComponentBuilder> Components
in place patch would be `.Select(x => x.Build())` when getting and `.Select(x => x.ToBuilder())` when setting